src/components/editor/ResourceTemplateForm.jsx

```diff
@@ -85,7 +84,11 @@ export class ResourceTemplateForm extends Component {
 
   resourceTemplatePromises = async (templateRefs) => {
```

Comment:
so now this is returning a single promise, yes?   Would `resourceTemplatesPromise` be more accurate?

Reply:
I don't think so...It's doing Promise.all and iterating over the collected resource templates that it found in the profile.

Reply:
yes, but it returns a single promise (I had to look that up) for all of the templates combined.  Merging, but if you agree, feel free to make a tiny PR.

Reply:
ok that makes sense.